### PR TITLE
No_SA_ID_Year_Entry, No_SA_ID_Month_Entry, No_SA_ID_Day_Entry tests

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
@bruskiza ready for review

File: go-app.js [hunk 1]
```diff
@@ -203,22 +213,6 @@ go.app = function() {
...
-      return new FreeText(name, {
-        question: question,
-        check: function(content) {
-          if (true) {
-            return null; // vumi expects null or undefined if check passes
-          } else {
-            return error;
-          }
-        },
-        next: 'No_SA_ID_Gender_Entry'
```
Comment (smn) [inline go-app.js:218]: I fixed this above, it now goes to the day after asking the month. The Day state wasn't asked anywhere.

File: go-app.js [hunk 2]
```diff
@@ -193,7 +203,7 @@ go.app = function() {
     });
     self.states.add('No_SA_ID_Gender_Entry', function(name) {
       return new ChoiceState(name, {
-        question: $("Please select the patient’s gender:"),
+        question: $("Please select the patient's gender:"),
```
Comment (smn) [inline go-app.js:206]: Note the change of the `’` to `'`

**Comment by bruskiza** [comment]
:+1: thanks!

**Pull request merged**